### PR TITLE
Add bug report and feature request GitHub issue templates

### PR DESCRIPTION
**Pull request by jon-wei**
This PR suggests template formats for bug reports and feature requests, inspired somewhat by the Kafka issue structure: https://cwiki.apache.org/confluence/display/KAFKA/Reporting+Issues+in+Apache+Kafka

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: A template for Druid bug reports
+title: "[BUG REPORT]"
```
[the remark this answers is not in the document]
Comment (jon-wei) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Removed the [BUG REPORT] title

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+The Druid version where the bug was encountered.
+
+# Description
+
+Please include as much detailed information about the bug as possible.
+- Cluster size
+- Configurations in use
+- Steps to reproduce the bug
+- The error message or stack traces encountered. Providing more context, such as nearby log messages or even entire logs, can be helpful.
+- Any debugging that the issue submitter has already done
```
[the remark this answers is not in the document]
Comment (jon-wei) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Changed

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+about: A template for Druid bug reports
+title: "[BUG REPORT]"
+labels: Bug
+assignees: ''
+
+---
+
+# Summary
```
[the remark this answers is not in the document]
Comment (jon-wei) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Removed

**Comment by jon-wei** [comment]
I added a feature request template as well

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+name: Feature/Change Request
+about: A template for Druid feature and change requests
+title: "[REQUEST]"
```
Comment (gianm) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Nit, but, I'd remove the brackets from this one too.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+labels: Wish list
+assignees: ''
+
+---
+
+# Description
+
+Please describe the feature or change with as much detail as possible.
```
Comment (gianm) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Maybe add something about how if you have a detailed implementation in mind, and you're wanting to contribute it yourself, "Proposal" is better than "Feature request".

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 4 above)
Comment (jon-wei) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Removed

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 5 above)
Comment (jon-wei) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Added a note on proposals

**Comment by gianm** [comment]
I'll merge this without TC passing - it seems broken and will need to look into that separately. The things it's complaining about are not relevant to this PR, which doesn't change code.

**Pull request merged**